src/NuGet.Clients/NuGet.CommandLine/Commands/InstallCommand.cs

```diff
@@ -203,6 +206,38 @@ private CommandLineSourceRepositoryProvider GetSourceRepositoryProvider()
             return new CommandLineSourceRepositoryProvider(SourceProvider);
         }
 
+        private DependencyBehavior TryGetDependencyBehavior(string behaviorStr)
+        {
+            DependencyBehavior dependencyBehavior;
+
+            if (!Enum.TryParse<DependencyBehavior>(behaviorStr, /* ignoreCase */ true, out dependencyBehavior))
+            {
+                throw new CommandLineException(string.Format(CultureInfo.CurrentCulture, LocalizedResourceManager.GetString("InstallCommandUnknownDependencyVersion"), behaviorStr));
+            }
+
+            return dependencyBehavior;
+        }
+
+        private DependencyBehavior GetDependencyBehavior()
+        {
+            // If dependencyVersion is not set by either the config or commandline, default dependency behavior is 'Lowest'.
+            DependencyBehavior dependencyBehavior = DependencyBehavior.Lowest;
+
+            string settingsDependencyVersion = SettingsUtility.GetConfigValue(Settings, "dependencyVersion");
+
+            // Check to see if commandline flag is set. Else check for dependencyVersion in .config.
+            if (!string.IsNullOrEmpty(DependencyVersion))
```

Comment:
DependencyBehavior set from commandline should take precedence over everything else.